SU-HSDO/suhumsci
# Arch 88 fix global menu styles

# READY

# Summary
- Fixes main menu for desktop breakpoints, adding padding and justifying to the right.

# Needed by end of sprint

# Urgency
- Must Fix, Low Priority

# Steps to Test

1. Visit home page
2. Confirm that menu looks good at desktop breakpoints (600px and up)

# Affected Projects or Products
- No impact on projects, products, or modules outside the Archaeology theme.

# Associated Issues and/or People
- JIRA ticket ARCH-88
- Other PRs
